azurerm/resource_arm_traffic_manager_profile.go

```diff
@@ -316,6 +341,18 @@ func resourceAzureRMTrafficManagerMonitorConfigHash(v interface{}) int {
 		if v, ok := m["path"]; ok && v != "" {
 			buf.WriteString(fmt.Sprintf("%s-", m["path"].(string)))
 		}
+
+		if v, ok := m["interval_in_seconds"]; ok && v != "" {
```

Comment:
As you are changing the hash function you will need to add a state migration function & test.